From 0f72ff67cf029087b7502007537505543ad161c2 Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyfox@gentoo.org>
Date: Sun, 20 Nov 2011 15:58:40 +0300
Subject: [PATCH] SPL.cabal: remove haskell98 package from Haskell2010 programs

Linking setup ...
Configuring SPL-0.0.0.920...
setup: At least the following dependencies are missing:
haskell98 ==1.*

Signed-off-by: Sergei Trofimovich <slyfox@gentoo.org>
---
 SPL.cabal |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/SPL.cabal b/SPL.cabal
index ca23d56..7b01613 100644
--- a/SPL.cabal
+++ b/SPL.cabal
@@ -32,13 +32,13 @@ Test-Suite spl-test-hunit
 Executable	spli
   Main-Is:	SPLI.hs
   default-language: Haskell2010
-  Build-Depends:  base == 4.*, containers == 0.*, HUnit == 1.*, bytestring == 0.*, hoopl >= 3.8.7.1, mtl == 2.*, parsec >= 3.1, directory == 1.*, QuickCheck == 2.*, haskell98 == 1.*
+  Build-Depends:  base == 4.*, containers == 0.*, HUnit == 1.*, bytestring == 0.*, hoopl >= 3.8.7.1, mtl == 2.*, parsec >= 3.1, directory == 1.*, QuickCheck == 2.*
   Other-Modules:   Bar
 
 Executable	splm
   Main-Is:	M.hs
   default-language: Haskell2010
-  Build-Depends:  base == 4.*, containers == 0.*, HUnit == 1.*, bytestring == 0.*, hoopl >= 3.8.7.1, mtl == 2.*, parsec >= 3.1, directory == 1.*, QuickCheck == 2.*, haskell98 == 1.*
+  Build-Depends:  base == 4.*, containers == 0.*, HUnit == 1.*, bytestring == 0.*, hoopl >= 3.8.7.1, mtl == 2.*, parsec >= 3.1, directory == 1.*, QuickCheck == 2.*
   Other-Modules:   Bar
 
 Executable     spl-test-adhoc-hn
-- 
1.7.3.4

